### PR TITLE
Issue #206: Activated at risk section with initial entries

### PR DESCRIPTION
**Pull request by jdsmith3000**
Also listed MediaKeySession Destroyed, since it is related.

### CODE REVIEW

File: encrypted-media-respec.html [hunk 1]
```diff
@@ -134,15 +134,14 @@
       <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/192">Issue 192</a> - The specification should do more to encourage/ensure CDM-level interoperability.</p>
       <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/221">Issue 221</a> - This specification contains sections for describing <a href="#security">security</a> and <a href="#privacy">privacy</a> considerations. These sections are not final and review is welcome.</p>
 
-<!-- This will be populated when addressing https://github.com/w3c/encrypted-media/issues/206.
       <p>The following features are <strong>at risk</strong> and may be removed due to lack of implementation.
```
Comment (ddorwin) [inline encrypted-media-respec.html]: I don't think "lack of implementation" covers all possible reasons for the items being discussed in #206. Should we just drop everything after "removed?"

File: encrypted-media-respec.html [hunk 2]
```diff
@@ -134,15 +134,14 @@
...
       </p>
       <ul>
-        <li><a def-id=""></a></li>
+        <li><a def-id="persistent-usage-record-session"></a></li>
```
Comment (ddorwin) [inline encrypted-media-respec.html]: I suggest providing a bit more context. For example:
`The <a def-id="persistent-usage-record-session"></a>` session type`

nit: I believe we (generally) use `<p>` inside `<li>` now.

File: encrypted-media-respec.html [hunk 3]
```diff
@@ -134,15 +134,14 @@
...
+        <li><a def-id="media-key-session-destroyed-algorithm"></a></li>
```
Comment (ddorwin) [inline encrypted-media-respec.html]: Similarly, `The <a def-id="media-key-session-destroyed-algorithm"></a> algorithm`

File: encrypted-media-respec.html [hunk 4]
```diff
@@ -134,13 +134,15 @@
...
-      <p>The following features are <strong>at risk</strong> and may be removed due to lack of implementation.
-      </p>
+      <p>The following features are <strong>at risk</strong> and may be removed:
+	  </p>
       <ul>
-        <li><a def-id="persistent-usage-record-session"></a></li>
-        <li><a def-id="media-key-session-destroyed-algorithm"></a></li>
+        <li><p>The <a def-id="persistent-usage-record-session"></a> session type.</p></li>
+        <li><p>The <a def-id="media-key-session-destroyed-algorithm"></a> algorithm.</p></li>
+        <li><p>Setting the <var>media element</var>'s <a def-id="readystate"></a> value based on key availability in the <a def-id="resume-playback-algorithm"></a> algorithm.</p></li>
```
Comment (ddorwin) [inline encrypted-media-respec.html]: We need to cover both "directions":
`<a def-id="queue-waitingforkey-algorithm"></a> and <a def-id="resume-playback-algorithm"></a> algorithms.`

File: encrypted-media-respec.html [hunk 5]
```diff
@@ -134,13 +134,15 @@
...
+        <li><p>The <a def-id="are-insecure-contexts-allowed-algorithm"></a> algorithm and steps that call it.</p></li>
```
Comment (ddorwin) [inline encrypted-media-respec.html]: Should we be more clear about the impact? For example, "Support for insecure contexts, including the ..."

File: encrypted-media-respec.html [hunk 6]
```diff
@@ -134,13 +134,15 @@
...
       </ul>
-
+
```
Comment (ddorwin) [inline encrypted-media-respec.html]: nit: whitespace

File: encrypted-media-respec.html [hunk 7]
```diff
@@ -134,13 +134,15 @@
       <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/192">Issue 192</a> - The specification should do more to encourage/ensure CDM-level interoperability.</p>
       <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/221">Issue 221</a> - This specification contains sections for describing <a href="#security">security</a> and <a href="#privacy">privacy</a> considerations. These sections are not final and review is welcome.</p>
 
-      <p>The following features are <strong>at risk</strong> and may be removed due to lack of implementation.
-      </p>
+      <p>The following features are <strong>at risk</strong> and may be removed:
+	  </p>
```
Comment (ddorwin) [inline encrypted-media-respec.html]: nit: Incorrect indent. Actually, though, we can just move it to the previous line since it's a single sentence.

**Comment by jdsmith3000** [comment]
Made all requested changes.

**Comment by ddorwin** [comment]
LGTM. I modified the summary, which will hopefully be used in the commit message. Be sure to delete the various commit messages when merging.

**Pull request merged**